controllers/discovery/initDiscoveryUserData.js

```diff
@@ -118,10 +120,13 @@ const InitDiscoveryUserData = async (req, res) => {
         LIMIT :limit
         OFFSET :offset`;
 
+    console.log('topicsIds', topicIds);
+
     let usersWithCommonFollowerResult = await sequelize.query(usersWithCommonFollowerQuery, {
       type: QueryTypes.SELECT,
       replacements: {
         userId,
+        // topicIds: topicIds || [],
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Logging `topicIds` can be useful for debugging purposes, but it's generally not a good practice to leave console logs in production code. Consider removing this line or using a proper logging mechanism that can be toggled on/off based on the environment (development/production).
```diff
- console.log('topicsIds', topicIds);
```
In the replacements object, the commented out line `// topicIds: topicIds || [],` seems to be a previous attempt at handling null or empty `topicIds`. Since you've already handled this case with the `topicIdQuery`, this line can be safely removed.
```diff
- // topicIds: topicIds || [],
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->